must-gather/collection-scripts/gather

```diff
@@ -20,6 +20,9 @@ resources+=(performanceprofile)
 # machine/node resources
 resources+=(nodes machineconfigs machineconfigpools featuregates kubeletconfigs tuneds)
 
+# pods information
+resources+=(pods)
```

Comment:
does the base must-gather image collect this info?

Reply:
Not really they get pod information from some namespaces but not all of them and to make accurate checks we need to know all the "guaranteed" pods in order to be able to calculate all the exclusively assigned cpus.

Reply:
Not exactly just `openshift-*` namespaces but a list of namespaces ([see](https://github.com/openshift/must-gather/blob/87c1fd3aee6a6ec7359406a14e1ad0d8d1f8784e/collection-scripts/gather#L28)

As far as I understand we need all the exclusively assigned CPUs to intersect them with the IRQs affinity lists, and to get them we need to know at least all the containers with an integer CPU request in any "guaranteed" pod.

Reply:
named_resources+=(ns/default ns/openshift ns/kube-system ns/openshift-etcd)
https://github.com/openshift/must-gather/blob/master/collection-scripts/gather#L28